src/platform/site-wide/sass/shame.scss

```diff
@@ -363,7 +363,7 @@ $medium-phone-screen: 375px;
 
         .va-dropdown {
           .va-dropdown-trigger {
-            background-position: right 0.8rem center;
+            background-position: right 8px center;
```

Comment:
Should this be in `rem`?

Reply:
@randimays you're right - this style is not used on va.gov so it doesn't have to be converted. I have pushed a commit.